app/assets/javascripts/discourse/app/components/composer-editor-uppy.js

```diff
@@ -0,0 +1,7 @@
+import ComposerEditor from "discourse/components/composer-editor";
+import ComposerUploadUppy from "discourse/mixins/composer-upload-uppy";
+
+export default ComposerEditor.extend(ComposerUploadUppy, {
+  layoutName: "components/composer-editor",
```

Comment:
should use layout and not layoutName, look at what I do in select-kit in the codebase

Reply:
I tried this with these combinations and it did not work, it seems like select-kit is the only thing successfully doing it this way?:

* import layout from "discourse/app/templates/components/composer-editor";
* import layout from "templates/components/composer-editor";
* import layout from "discourse/templates/components/composer-editor";
* import layout from "discourse/components/composer-editor";

We use `layoutName` in many other places in the project:

![image](https://user-images.githubusercontent.com/920448/129111453-df3fd6bf-410c-4edc-b228-91d8c4861eb0.png)

I will leave it as this for now, don't want to sink a lot of time into figuring out why it doesn't load right when I can just use `layoutName`. Please let me know if you know what the correct path should be.

Reply:
Sorry should have posted the error, it is a "could not find module" error every time:

![image](https://user-images.githubusercontent.com/920448/129111559-a3b834d5-232a-4a71-879d-ce33711be427.png)

Reply:
Yes I know we are still doing it that way in some places, will look at this once it's merged, no hurries.